ansible/playbooks/tasks/aws-cluster-bootstrap.yaml

```diff
@@ -0,0 +1,404 @@
+---
+# This cluster-bootstrap tais an attempt of creating a standard way to build
```

Comment:
s/tais/this/

Reply:
Maybe it is not `this`, I don't know why use `tais`.

Reply:
I think it's:
```
# This cluster-bootstrap playbook is an attempt of creating a standard way to build